Userland/Libraries/LibJS/Runtime/AtomicsObject.cpp

```diff
@@ -225,7 +225,7 @@ static ThrowCompletionOr<Value> atomic_compare_exchange_impl(VM& vm, TypedArrayB
     // 9. Let elementSize be TypedArrayElementSize(typedArray).
 
     // 10. Let isLittleEndian be the value of the [[LittleEndian]] field of the surrounding agent's Agent Record.
-    constexpr bool is_little_endian = __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__;
+    constexpr bool is_little_endian = AK::HostIsLittleEndian;
```

Comment:
There's some other `__BYTE_ORDER__` checks in LibAudio and LibCrypto, should they be changed as well?

Reply:
I've already changed the ones that are worth changing, the LibCrypto one picks between two different constexpr functions, I'd rather not mess with that.

Reply:
(For reference, I _did_ change the LibAudio one)